src/OrchardCore/OrchardCore/Modules/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -272,6 +273,71 @@ private static void AddAntiForgery(OrchardCoreBuilder builder)
             });
         }
 
+        /// <summary>
+        /// Adds backwards compatibility to the handling of SameSite cookies.
+        /// </summary>
+        public static void AddSameSiteCookieBackwardsCompatibility(OrchardCoreBuilder builder)
+        {
+            builder.ConfigureServices(services =>
+                {
+                    services.Configure<CookiePolicyOptions>(options =>
+                    {
+                        options.MinimumSameSitePolicy = AspNetCore.Http.SameSiteMode.Unspecified;
```

Comment:
Add some using love? 😄

Reply:
;)

Reply:
No can't do, it conflicts with a different namespace.

Reply:
Is the conflicting class coming from `Microsoft.Net.Http`?
Anyway, you can still do `using SameSiteMode = Microsoft.AspNetCore.Http.SameSiteMode` to make things nicer.

Reply:
Ok, will do tonight.